src/index.js

```diff
@@ -407,12 +408,14 @@ class CopyPlugin {
     logger.log(`begin globbing '${glob}'...`);
 
     /**
-     * @type {GlobEntry[]}
+     * @type {string[]}
```

Comment:
whats the typedef change here?

Reply:
`objectMode: true` changes the return type from the default `string[]` to `GlobEntry[]`. Since we've removed `objectMode: true` we can go back to `string[]`, which is simpler and has better performance